samples/subsys/usb/hid-mouse/Kconfig

```
@@ -8,6 +8,6 @@ config USB_DEVICE_PID
 	default USB_PID_HID_MOUSE_SAMPLE
 
 config USB_DEVICE_REMOTE_WAKEUP
-	default SOC_NRF52840
+	default y if (CONFIG_USB_NRF52)
```

This is incorrect, `CONFIG_USB_NRF52` is not a Kconfig option.
```suggestion
	default USB_NRF52
```